docs/usage/quickstart.rst

```diff
@@ -84,11 +84,17 @@ See :ref:`usage-indexing`.
 Accessing the contents
 ----------------------
 
-You can use ``hist.view()`` to get
-a Numpy array (or a RecArray-like wrapper for non-simple storages).
-Most methods like ``.view()`` offer an optional keyword
-argument that you can pass, ``flow=True``, to enable the under and
-overflow bins (disabled by default).
+You can use ``hist.values()`` to get a Numpy array from any histogram. You can
+get the variances with ``hist.variances()``, though if you fill an unweighted
+storage with weights, this will return None, as you no longer can compute the
+variances correctly (please use a weighted storage if you need to). You can
+also get the effective counts with ``.counts()``; this will return counts even
```

Comment:
Again please replace "effective counts" with just "counts" and then add an explanation what effective counts are in case of a weighted histogram or better just refer to one detailed explanation somewhere in the docs. Nobody understands what effective counts are unless they are familiar with the concept, and many people are not familiar with it.

Reply:
We need to make sure we cross link a bit more, and yes. :)

Reply:
Holding this open till I set up some cross linking.